engine/src/main/java/org/pentaho/di/trans/steps/delete/Delete.java

```diff
@@ -54,7 +54,7 @@ public class Delete extends BaseStep implements StepInterface {
   private DeleteData data;
 
   public Delete( StepMeta stepMeta, StepDataInterface stepDataInterface, int copyNr, TransMeta transMeta,
-    Trans trans ) {
+                 Trans trans ) {
```

Comment:
Not sure why this formatting needed to change; this doesn't look aligned with our code standards.